### ISSUE TITLE
Lack of detail on why a transaction failed

### ISSUE THREAD

**Issue by Justinzobel** [rpm-software-management/dnf5#538]
```
dnf5 --version
dnf5 version 5.0.6
dnf5 plugin API version 1.0
libdnf5 version 5.0.6
libdnf5 plugin API version 1.0

Loaded dnf5 plugins:
  name: builddep
  version: 1.0.0
  API version: 1.0

  name: changelog
  version: 1.0.0
  API version: 1.0
  ```
  
  
dnf5 error message: `Rpm transaction failed.`
  
dnf error message:
```
Error Summary
Disk Requirements:
At least 1722MB more space needed on the / filesystem. 
```

**Comment by jan-kolarik** [comment]
Hi, thanks for the report! Could you please add more details, like a reproducer for this? It seems like a similar problem which was fixed recently: https://github.com/rpm-software-management/dnf5/issues/515

**Comment by Justinzobel** [comment]
No worries. No real direct way to reproduce. I chrooted into an img file that was mounted in a loop device. I didn't have enough space. You could reproduce this by trying to install enough packages that your disk would fill up if it executed the transaction.

**Comment by jan-kolarik** [comment]
I believe this was fixed in dnf5 5.0.10 release by this [PR](https://github.com/rpm-software-management/dnf5/pull/517) as the `Rpm transaction failed.` error message should be always accompanied by the RPM problems output as looking into the sources.

**Comment by Justinzobel** [comment]
Hopefully I can try that in F38 soon! 🙂

**Comment by jan-kolarik** [comment]
Probably soon: https://bodhi.fedoraproject.org/updates/FEDORA-2023-c8fb79e192 :)

**Comment by jan-kolarik** [comment]
Should be fixed in `dnf5-5.0.11-2.fc38` which is already available in Fedora 38.
